coin_flip/viewer.html

```diff
@@ -0,0 +1,123 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <!-- A real extension would not be able to link to external CDNs; we just do it here for simplicity -->
+    <link rel= "stylesheet" type = "text/css" href = "./Coinflipstyle.css">
+    <!--
+      Twitch only allows loading assets from certain white-listed domains. Above we reference a font
+      from Google"s font CDN. Here we load the latest version of Muxy"s JavaScript SDK. You can
+      reference specific versions by replacing `latest` with a version number (e.g. `2.0.0`), or keep
+      this as-is to automatically load the most recent version.
+    -->
+    <script src="https://ext-cdn.muxy.io/medkit/latest/medkit.umd.js"></script>
+
+    <!--
+      Here is the entire code block needed to power the panel. It uses Muxy"s state loading to get
+      the last sent message, and easy event binding to get notified when a new message is sent.
+    -->
+    <script type="text/javascript">
+    
+      /*
+      * Here we are giving the SDK a hint that this will run on the broadcaster"s dashboard and
+      * have the same rights a broadcaster does. This is only needed and used when developing an
+      * extension. Once it is deployed to production, the backend will use the correct rights from
+      * the extension system. You can leave this line in and it will not adversely affect the
+      * final extension.
+      */
+      Muxy.testJWTRole = "viewer";
+
+      Muxy.setup({ clientID: "replaceme" });
+      const sdk = new Muxy.SDK();
+
+      /*
+      * Here we are using lodash to throttles calls to this function, so if it is called more than once in a short period of
+      * time, only one instance will be called.
+      */
+      let choice = null;
+
+      window.determineWinner = function() {
+        var r = document.querySelector("#choices");
+        let p = document.querySelector("#results");
+        let RandomNumber = Math.round(Math.random()); // Generating random number to determine if heads or tails.
+        if(RandomNumber == choice){
+          p.innerText = "You guessed right, Winner!";
+          sdk.rank("WinLose", "win");
+        } else {
+          p.innerText = "You guessed wrong, Loser!";
+          sdk.rank("WinLose", "lose");
+        }
+        //Here we determine whether or not your choice matches the correct answer.
+
+        if(choice == 1) {
+          r.innerText = "You picked Heads!";
+        } else {
+          r.innerText = "You picked Tails!";
+        }
+        //Here we determine what your choice is and display it back to you.
+        
+        document.querySelector("#results").style.opacity = 1;
+        document.querySelector("#choices").style.opacity = 1;
+        //This is where we disable buttons and make them disappear.
```

Comment:
Comments should appear before the code they describe. Also comment here is incorrect.